app/helpers/insured/families_helper.rb

```diff
@@ -263,7 +263,7 @@ def enrollment_state_label(enrollment)
     label = group[condition] || { text: enrollment.aasm_state.to_s.titleize, color: 'grey' }
     # Coverage reinstated is a special case where the aasm state is something else but we want to show it as reinstated in "green" (active) scenarios
     label = state_groups[:coverage_reinstated][:default] if enrollment.is_reinstated_enrollment? && label[:color] == 'green'
-    content_tag(:span, label[:text], class: "label label-#{label[:color]}")
```

Comment:
should the px-2 class be behind the feature flag?

Reply:
probably, but this is all going to get reworked with the enrollment tile ticket - this change was just to make thing a little more functional until then and it barely changes the exisitng/non-bs UI